tests/conditional_compilation/conftest.py

```diff
@@ -99,4 +113,59 @@ def artifacts(request):
 @pytest.fixture(scope="session")
 def openvino_root_dir(request):
     """Fixture function for command-line option."""
-    return request.config.getoption("openvino_ref")
+    openvino_root_dir = request.config.getoption("openvino_root_dir")
+    validate_path_arg(openvino_root_dir, "openvino_root_dir", is_dir=True)
+
+    return openvino_root_dir
+
+
+@pytest.fixture(scope="session")
```

Comment:
looks loke scope should be `function`. Also note that raising exceptions not from `function` scope may lead to crash of full run

Reply:
Discussed offline. No reason to change scope due it should be computed once per session, not per function